cmd/traffic/cmd/manager/service.go

```diff
@@ -149,6 +186,9 @@ func (s *service) ArriveAsClient(ctx context.Context, client *rpc.ClientInfo) (*
 	}
 
 	installId := client.GetInstallId()
+
+	s.metrics.ConnectCounter.With(prometheus.Labels{"client": client.Name, "install_id": client.InstallId}).Inc()
```

Comment:
Maybe I would just attach methods to that metric object so it's cleaner to read for all functions

Like s.metrics.OnConnect(client.Name, client.InstallID)